sycl/unittests/Extensions/CommandGraph/Queries.cpp

```diff
@@ -110,7 +110,7 @@ TEST_F(CommandGraphTest, NodeTypeQueries) {
 
   auto NodeMemset = Graph.add(
       [&](sycl::handler &cgh) { cgh.memset(PtrB, 7, 16 * sizeof(int)); });
-  ASSERT_EQ(NodeMemset.get_type(), experimental::node_type::memset);
+  ASSERT_EQ(NodeMemset.get_type(), experimental::node_type::memfill);
```

Comment:
We're returning the wrong node type from the user perspective here, it should be a `memset` because that's what the users asked. Using a memfill underneath is an implementation detail. See my other comment.